config/secrets.yml

```diff
@@ -12,7 +12,8 @@
 
 development:
   secret_key_base: ed2bbe63fc01ef3a21654d567cbe221d50129e4e84b12b328b0c3b5bb294a04b487db8bd93316e2c3417f34b734ffd31ae987208b03bd736d9cd0ebcd9634b34
-
+  aws_access_key_id: AKIAIMMZNI5YBHU4WBYA
+  aws_secret_access_key: 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
```

Comment:
These two need to be removed in favor of the same `<%= ENV["AWS_*"] %>` variables in staging and production. Access keys, etc., should never be stored in your repository. And if I did that, I shouldn't have.